README.md

```diff
@@ -1,13 +1,8 @@
 # Project Movies
 
-Replace this readme with your own information about your project.
+A site where you can scroll through the latest popular movies. Built in React using API and react router. By Jenny Fiskaare and Linnea Frisk.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://project-movies-linnea-and-jenny.netlify.app/
```

Comment:
Your page looks absolutely on point to the design provided for this week. I especially like the hover effect to darken each movie and how it transitions. Really great job accomplishing the design this week! :)